hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/key/AbstractOMKeyDeleteResponse.java

```diff
@@ -115,15 +115,15 @@ protected void addDeletionToBatch(
    *  file table (which is in prefix format) and adds the fullKey
    *  into the deletedTable
    * @param keyName     (format: objectId/key)
-   * @param fullKeyName (format: vol/buck/key)
+   * @param deleteKeyName (format: vol/buck/key)
```

Comment:
```suggestion
   * @param deleteKeyName (format: vol/buck/key/objectId)
```
Please also update `OmMetadataManagerImpl` with the schema change.

Reply:
Updated...